src/FsAutoComplete.Core/Project.fs

```diff
@@ -23,23 +83,47 @@ type Project (projectFile, onChange: ProjectFilePath -> unit) =
                 r.Reply response
                 return! loop (lastWriteTime, response)
             | SetResponse r ->
-                return! loop (lastWriteTime, r)
-        }    
-        loop (File.GetLastWriteTimeUtc projectFile, None)
+                persistentCache.SaveCache(lastWriteTime, r)
 
-    let fullPath = Path.GetFullPath projectFile
+                return! loop (lastWriteTime, r)
+        }
+        let lwt =
+            if File.Exists projectAsset then
+                let a = File.GetLastWriteTimeUtc projectAsset
+                let p = File.GetLastWriteTimeUtc projectFile
+                if a > p then a else p
```

Comment:
nope. i was pretty sure normal `max` handled it, but seems not.
Something for an RFC...